### PR TITLE
fix(an): fixing few minor errors

### PR DESCRIPTION
**Pull request by sigruntg**
# Few minor errors

[1nbj6t4](https://app.clickup.com/t/1nbj6t4)
[1nbj5vp](https://app.clickup.com/t/1nbj5vp)
[1nbj68q](https://app.clickup.com/t/1nbj68q)
[1nbj5nz](https://app.clickup.com/t/1nbj5nz)

## What

* Change text that reviewer sees in overview.
* Fix messages.
* Add border radius to bottom of red border in application status.
* Remove information sign in logo.

## Why

Fixing from QA.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/accident-notification/src/lib/messages/applicantInformation.ts [hunk 1]
```diff
@@ -12,6 +12,11 @@ export const applicantInformation = {
       defaultMessage: 'Upplýsingar um þig',
       description: 'Information about you',
     },
+    titleForReviewer: {
```
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/lib/messages/applicantInformation.ts:15]: This already exists in the file in line 61 forTheThirdParty

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
Just needs a minor change to pass

File: libs/application/templates/accident-notification/src/lib/messages/applicantInformation.ts (hunk 1 above)
Comment (sigruntg) [reply libs/application/templates/accident-notification/src/lib/messages/applicantInformation.ts]: Fixed!

File: libs/application/templates/accident-notification/src/fields/ApplicationStatus/StatusStep/index.tsx [hunk 2]
```diff
@@ -71,6 +71,7 @@ export const StatusStep: FC<ReviewSectionProps> = ({
           justifyContent="spaceBetween"
           background="roseTinted100"
           padding={4}
+          className={cn(styles.bottomBorderRadius)}
```
Comment (Thorrik58) [inline libs/application/templates/accident-notification/src/fields/ApplicationStatus/StatusStep/index.tsx:74]: Dont think you need cn as we arent joining classnames together here, do you?

Comment (sigruntg) [reply libs/application/templates/accident-notification/src/fields/ApplicationStatus/StatusStep/index.tsx]: Propably not, removed 😎

**Review by Thorrik58 — APPROVED** [review]

**Pull request merged**